modules/apps/27-interchain-accounts/controller/ibc_middleware.go

```diff
@@ -176,6 +176,61 @@ func (im IBCMiddleware) OnTimeoutPacket(
 	return im.app.OnTimeoutPacket(ctx, packet, relayer)
 }
 
+// OnChanUpgradeInit implements the IBCMiddleware interface
+func (im IBCMiddleware) OnChanUpgradeInit(
+	ctx sdk.Context,
+	order channeltypes.Order,
+	connectionHops []string,
+	portID, channelID string,
```

Comment:
nit: I'd rather have these on separate lines but deferring to @damiannolan

Reply:
I can push an update to this PR :)